core/state/statedb_test.go

```diff
@@ -979,7 +979,8 @@ func testMissingTrieNodes(t *testing.T, scheme string) {
 	)
 	if scheme == rawdb.PathScheme {
 		tdb = triedb.NewDatabase(memDb, &triedb.Config{PathDB: &pathdb.Config{
-			CleanCacheSize:  0,
+			TrieCleanSize:   0,
+			StateCleanSize:  0,
 			WriteBufferSize: 0,
```

Comment:
Should we also disable the background generation (SnapshotNoBuild: true) here? Seems like disabling it does not impact the outcome of the test and we would save some resources

Reply:
Same with a few other places where we use `pathdb.Defaults`, disabling the background generation there, saved around 200ms in my (very rough) tests

Reply:
Okay nvm, the 200ms was just a fluke on my machine, seem to be almost equal. So probably not worth it